lib/ublksrv.c

```diff
@@ -858,6 +858,11 @@ static int ublksrv_reap_events_uring(struct io_uring *r)
 	return count;
 }
 
+int ublksrv_queue_reap_events(struct io_uring *r)
```

Comment:
I meant using "struct ublksrv_queue *" as the API's argument, instead of "struct io_uring *" because all queue APIs takes "struct ublksrv_queue *" as its parameter.

Reply:
ok, done